docs/how-to/code/chisel-existing-rock/chiselled-rock/rockcraft.yaml

```diff
@@ -0,0 +1,15 @@
+name: chiselled-python
+base: bare  # using a bare base
+build-base: ubuntu@22.04  # using Ubuntu Jammy only for the build environment
+version: "3.11"
+summary: Chiselled Python3.11 rock
+description: Example of a chiselled Python rock
+platforms:
+  amd64:
+run-user: _daemon_  # setting a non-root user for the rock
+parts:
+  install-python-slices:
+    plugin: nil
+    stage-packages:
+      - base-files_release-info # we only want the release information
```

Comment:
@cjdcordeiro doesn't python3.11_core depend on base-files_release-info. Do we need to explicitly declare it?

Reply:
`base-files` is one of those that due to being an essential, it isn't always listed as a pkg dependency. in this case, it's not even needed for python to run. the `release-info` slices only brings in stuff like `/etc/os-release` which is a must for allowing 3rd tools to audit and scan the resulting image